airbyte-webapp/src/components/ui/CodeEditor/CodeEditor.tsx

```diff
@@ -40,14 +36,14 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
       base: "vs-dark",
       inherit: true,
       rules: [
-        { token: "string", foreground: expandHexValue(styles.darkString) },
-        { token: "string.yaml", foreground: expandHexValue(styles.darkString) },
-        { token: "string.value.json", foreground: expandHexValue(styles.darkType) },
-        { token: "string.key.json", foreground: expandHexValue(styles.darkType) },
-        { token: "type", foreground: expandHexValue(styles.darkType) },
-        { token: "number", foreground: expandHexValue(styles.darkNumber) },
-        { token: "delimiter", foreground: expandHexValue(styles.darkDelimiter) },
-        { token: "keyword", foreground: expandHexValue(styles.darkKeyword) },
+        { token: "string", foreground: cssCustomPropToHex(styles.darkString) },
+        { token: "string.yaml", foreground: cssCustomPropToHex(styles.darkString) },
+        { token: "string.value.json", foreground: cssCustomPropToHex(styles.darkType) },
+        { token: "string.key.json", foreground: cssCustomPropToHex(styles.darkType) },
+        { token: "type", foreground: cssCustomPropToHex(styles.darkType) },
+        { token: "number", foreground: cssCustomPropToHex(styles.darkNumber) },
+        { token: "delimiter", foreground: cssCustomPropToHex(styles.darkDelimiter) },
+        { token: "keyword", foreground: cssCustomPropToHex(styles.darkKeyword) },
```

Comment:
Hey Joey, I just pushed up one small change in this commit: https://github.com/airbytehq/airbyte/pull/19344/commits/c9ce27b788d603a93b0678d03be977a93d801a21

Basically, we still need the logic to convert 3-character hex values to 6-character ones, because the Monaco editor throws an error when it receives 3-character values. 

This doesn't happen in local development mode, likely because the hex values are not being minified. But in production (i.e. when you gradle build the airbyte-webapp and go to `localhost:8000/connector-builder` instead of `localhost:3000/connector-builder`), then the hex values are shortened and monaco fails, which is why we need that expansion logic in the typescript code.

Reply:
Oops - I think I read the comment on `expandHexValues()` too hastily. Thanks for reverting that 👍